conn2res/plotting.py

```diff
@@ -70,3 +135,77 @@ def plot_performance_curve(df, title, num=2, figsize=(12, 10), savefig=False, bl
         fig.savefig(fname=os.path.join(FIG_DIR, f'{title}_score.png'),
                     transparent=True, bbox_inches='tight', dpi=300)
     plt.show(block=block)
+
+
+def plot_time_series(x, feature_set='orig', idx_ts=None, idx_features=None, n_features=None,
```

Comment:
The general idea behind this function is that we can call it multiple times when we want to create a figure. This helps a lot when we want to explore the data and see, for instance, how the inputs relate to the reservoir states etc.

If we want to plot on the same figure and/or subplot, we need to make sure to specify the same arguments (see num, subplot). We can also adjust the figure size (see figsize) depending on the number of suplots we want to have.

I tricked the way the legend works to be able to plot on the same subplot multiple times.